packages/version/README.md

```diff
@@ -386,6 +387,19 @@ lerna version --conventional-commits --create-release gitlab
 
 When run with this flag, `lerna version` will create an official GitHub or GitLab release based on the changed packages. Requires `--conventional-commits` to be passed so that changelogs can be generated.
 
+> **Note** to avoid creating too many "Version bump only for package x" when using independent mode, you could enable the option `--skip-bump-only-release`
+
+### `--skip-bump-only-release`
+
+When this option is enabled and a package version is only being bumped without any conventional commits detected, the GitHub/GitLab release will be skipped. This will avoid creating releases with only "Version bump only for package x" in the release notes, however please note that each changelog will still be updated the version bump only text.
```

Comment:
typo, it should be "are still going to be updated with the "version bump only" text."